src/commands/snapshot/compare.ts

```diff
@@ -50,8 +51,11 @@ export default class Compare extends SnapshotCommand {
         })
 
         if (updatedCommand) {
-          const {changedFlags} = this.diffCommandFlags(initialCommand.flags, updatedCommand.flags)
-          if (changedFlags.length > 0) {
+          const changedFlags = this.diffCommandProperty(initialCommand.flags, updatedCommand.flags).changedProperty
+          const changeAlias = this.diffCommandProperty(initialCommand.alias, updatedCommand.alias).changedProperty
+
+          if (changeAlias.length > 0 || changedFlags.length > 0) {
+            updatedCommand.alias = changeAlias
```

Comment:
Should we set `updatedCommand.alias` if there are no changed aliases but some changed flags?
```suggestion
          if (changeAlias.length > 0 || changedFlags.length > 0) {
            if (changeAlias.length > 0) updatedCommand.alias = changeAlias
```